WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/AttributeOptionListSelectorCommand.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import Yosemite
+
+/// Command to populate the shipping label paper size list selector
+///
+final class AttributeOptionListSelectorCommand: ListSelectorCommand {
+
+    enum Row: Equatable {
+        case anyAttribute
```

Comment:
Should this be `anyOption` or even `any`?

Reply:
I don't like `any` because it's too similar to Swift's `Any` when you quickly scan over the code. But I agree on `anyOption` 👍